python/doc/examples/data_analysis/manage_data_and_samples/plot_quantile_estimation_wilks.py

```diff
@@ -60,49 +30,82 @@
 inputDist.setDescription(["X1", "X2"])
 inputVector = ot.RandomVector(inputDist)
 
-# Create the output random vector Y=model(X)
+# Create the output random vector
 output = ot.CompositeRandomVector(model, inputVector)
 
 # %%
-# Quantile level
-alpha = 0.95
+# We define the order :math:`\alpha` of the quantile and the confidence level :math:`\beta`.
 
-# Confidence level of the estimation
+# %%
+alpha = 0.95
 beta = 0.90
 
 # %%
-# Get a sample of the variable
-N = 10**4
-sample = output.getSample(N)
-graph = ot.UserDefined(sample).drawCDF()
-view = viewer.View(graph)
+# We generate a sample of the variable.
+
+# %%
+n = 10**4
+sample = output.getSample(n)
+
+# %%
+# We get the empirical estimator of the quantile of order :math:`\alpha` which is the
+# :math:`\lfloor N\alpha \rfloor` -th order statistics evaluated on
+# the sample.
 
 # %%
-# Empirical Quantile Estimator
 empiricalQuantile = sample.computeQuantile(alpha)
+print(empiricalQuantile)
 
-# Get the indices of the confidence interval bounds
-aAlpha = ot.Normal(1).computeQuantile((1.0 + beta) / 2.0)[0]
-min_i = int(N * alpha - aAlpha * m.sqrt(N * alpha * (1.0 - alpha)))
-max_i = int(N * alpha + aAlpha * m.sqrt(N * alpha * (1.0 - alpha)))
-# print(min_i, max_i)
+# %%
+# The asymptotic confidence interval of level :math:`\beta` is :math:`\left[ X_{(i_n)}, X_{(j_n)}\right]`
+# such that:
+#
+# .. math::
+#
+#     i_n & = \lfloor n \alpha - \sqrt{n}a_{\beta}\sqrt{\alpha(1-\alpha)}\rfloor\\
+#     j_n & = \lfloor n \alpha + \sqrt{n}a_{\beta}\sqrt{\alpha(1-\alpha)}\rfloor
+#
+# where  :math:`a_{\beta}` is the quantile of order :math:`(1-\beta/2)` of the standard normal distribution.
+#
+# Then we have:
+#
+# .. math::
+#
+#     \lim\limits_{n \rightarrow +\infty} \Prob{x_{\alpha} \in \left[ X_{(i_n,n)}, X_{(j_n,n)}\right]} = 1-\beta
```

Comment:
```suggestion
#     \lim\limits_{\sampleSize \rightarrow +\infty} \Prob{x_{\alpha} \in \left[ X_{(i_\sampleSize,\sampleSize)}, X_{(j_\sampleSize,\sampleSize)}\right]} = 1-\beta
```

Reply:
Yes. I also changed the limi $1-\beta$ into $\beta$ as I modified the definition of $i_n$ and $j_n$ (following you remark)